satellite/satellitedb/overlaycache.go

```diff
@@ -509,6 +509,32 @@ func (cache *overlaycache) KnownUnreliableOrOffline(ctx context.Context, criteri
 	return badNodes, nil
 }
 
+// Reliable returns all reliable nodes.
+func (cache *overlaycache) Reliable(ctx context.Context, criteria *overlay.NodeCriteria) (nodes storj.NodeIDList, err error) {
+	// get reliable and online nodes
+	rows, err := cache.db.Query(cache.db.Rebind(`
+		SELECT id FROM nodes
+		WHERE disqualified IS NULL
+		  AND last_contact_success >= ? AND last_contact_success >= last_contact_failure`),
```

Comment:
Several other places in the code make similar queries, but use `>` for both of these comparisons instead of `>=`. Are you making it `>=` on purpose to solve something? If so, shouldn't we make this change in the other functions as well?

Reply:
This is using the negation of the other queries.

Reply:
Are you sure? The senses in which it is used seem to be the same. See e.g. the query from `KnownUnreliableOrOffline()`:

```
SELECT id FROM nodes
    WHERE id = any($1::bytea[])
    AND disqualified IS NULL
    AND last_contact_success > $2 AND last_contact_success > last_contact_failure
```